src/sentry/utils/pytest.py

```diff
@@ -96,13 +92,33 @@ def pytest_configure(config):
 
     settings.SOUTH_TESTS_MIGRATE = bool(os.environ.get('USE_SOUTH'))
 
+    if not hasattr(settings, 'SENTRY_OPTIONS'):
+        settings.SENTRY_OPTIONS = {}
+
+    settings.SENTRY_OPTIONS['redis.clusters'] = {
+        'default': {
+            'hosts': {
+                0: {
+                    'db': 9,
+                },
+            },
+        }
+    }
+
     # django mail uses socket.getfqdn which doesn't play nice if our
     # networking isn't stable
     patcher = mock.patch('socket.getfqdn', return_value='localhost')
     patcher.start()
 
-    client = StrictRedis(db=9)
-    client.flushdb()
+    from sentry.cache import default_cache
+    from sentry.options import default_store
+
+    default_store.cache = default_cache
```

Comment:
This should probably be factored into a method  in `runner.initializer` and follow the established pattern below.